viewer/jest.config.js

```diff
@@ -22,9 +22,6 @@ module.exports = () => {
       '\\.svg$': path.resolve(__dirname, './test-utilities/src/filetype-mocks/svgMock.js'),
       '\\.wasm$': path.resolve(__dirname, './test-utilities/src/filetype-mocks/wasmMock.js')
     },
-    globals: {
-      __webpack_public_path__: ''
-    },
```

Comment:
Now I'm really starting to wonder what this is :monocle_face:

Reply:
Path to CDN for V8 workers.